docs/UserGuide.md

```diff
@@ -131,9 +131,9 @@ Amigos provides functionality to manage logs, which are essentially detailed not
 
 Adds a log to an existing friend at the specified `INDEX` in Amigos.
 The `INDEX` refers to the index number shown in the displayed person's list.
-Format: `addlog INDEX t/[TITLE] d/[DESCRIPTION]`
+Format: `addlog INDEX ? n/NAME t/[TITLE] d/[DESCRIPTION]`
```

Comment:
Please edit the corresponding legend to explain what the `?` means, if not already done.